cache/vnode_test.go

```diff
@@ -0,0 +1,68 @@
+package cache
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestVnode_Create(t *testing.T) {
+	v := &Vnode{}
+	v.Create()
+
+	assert.NotNil(t, v.data, "Cache data map should not be nil")
+}
+
+func TestVnode_Destroy(t *testing.T) {
+	v := setupCache()
+	v.Destroy()
+
+	assert.Nil(t, v.data, "Cache instance should be nil")
+}
+
+func setupCache() *Vnode {
```

Comment:
I recommend making this function a testing helper function in go if test cases will call this function. It make it easier to understand the stack trace and output from printing when debugging. For more info read the [doc](https://pkg.go.dev/testing#T.Helper)
```suggestion
func setupCache(t *testing.T) *Vnode {
    t.Helper()
```